packages/ag-charts-community/src/chart/legend.ts

```diff
@@ -467,7 +468,7 @@ export class Legend extends BaseProperties {
             markerLabel.fontSize = fontSize;
             markerLabel.fontFamily = fontFamily;
 
-            const paddedSymbolWidth = this.updateMarkerLabel(markerLabel, datum, spriteDims);
+            const paddedSymbolWidth = this.updateMarkerLabel(this.spriteRenderer!, markerLabel, datum, spriteDims);
```

Comment:
nit: would using a for-loop instead of `Selection.each()` avoid the need to add `!`?

Reply:
From a risk perspective let's follow up on this distinctly, I think this is probably the minimalist change to remove the redundant allocation when there is no legend for the release.

Reply:
SGTM.